src/render/common.rs

```diff
@@ -80,6 +81,25 @@ impl Resolve for Text {
     }
 }
 
+impl Resolve for TranslatedText {
+    fn resolve<'t, 'py>(
+        &self,
+        py: Python<'py>,
+        template: TemplateString<'t>,
+        context: &mut Context,
+        _failures: ResolveFailures,
+    ) -> ResolveResult<'t, 'py> {
+        let resolved = Cow::Borrowed(template.content(self.at));
+        let django_translation = py.import("django.utils.translation")?;
+        let get_text = django_translation.getattr("gettext")?;
+        let resolved = get_text.call1((resolved,))?.extract::<String>()?;
```

Comment:
I would prefer to delegate to a Rust implementation here, but there's a whole lot of complexity in how Django finds and loads `.mo` files that we'd have to handle.